src/test/java/com/wikia/webdriver/common/templates/NewTestTemplate.java

```diff
@@ -8,37 +14,46 @@
 import com.wikia.webdriver.common.driverprovider.NewDriverProvider;
 import com.wikia.webdriver.common.driverprovider.UseUnstablePageLoadStrategy;
 
-import org.testng.SkipException;
-import org.testng.annotations.AfterMethod;
-import org.testng.annotations.BeforeMethod;
-
-import java.lang.reflect.Method;
-
 public class NewTestTemplate extends NewTestTemplateCore {
 
-  @BeforeMethod(alwaysRun = true)
-  public void start(Method method, Object[] data) {
-    Configuration.clearCustomTestProperties();
-    if (method.isAnnotationPresent(Execute.class)) {
-      if (!"".equals(method.getAnnotation(Execute.class).onWikia())) {
-        Configuration.setTestValue("wikiName", method.getAnnotation(Execute.class).onWikia());
-      }
-      if (!"".equals(method.getAnnotation(Execute.class).disableFlash())) {
-        Configuration
-            .setTestValue("disableFlash", method.getAnnotation(Execute.class).disableFlash());
-      }
+  private void setTestProperty(String key, String value) {
```

Comment:
There are 3 diferent types of properties in here:
1. Default
2. Local
3. Test

That is why the name should stay as it is